google-cloud-clients/google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -312,9 +312,11 @@ public void testLoggingHandler() throws InterruptedException {
     assertEquals(logId, entry.getLogName());
     assertEquals(ImmutableMap.of("levelName", "INFO",
```

Comment:
You don't need to change everything in the file, but I think we should update these so that every assert in this test uses Truth.